netlify.toml

```diff
@@ -0,0 +1,9 @@
+[build]
```

Comment:
I think configuration for `build` is not required.

Reply:
I myself am not sure about this, doing it for the first time.

Reply:
I'll remove the `[build]` part, and sent another commit then?

Reply:
@Arjun-sna Added another commit, removing `[[build]]` part.